### PR TITLE
Allows Checkbox to be checked separately 

### PR DESCRIPTION
**Pull request by jenessawhite**
This PR changes checkbox to allow just marking checked when the checkbox is selected and allows other actions to occur when selecting the container

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/dabit3/React-Native-Elements#readme",
   "dependencies": {
     "react-native-side-menu": "^0.20.1",
-    "react-native-tab-navigator": "^0.3.3"
+    "react-native-tab-navigator": "^0.3.3",
+    "react-native-vector-icons": "^4.0.0"
```
Comment (binoy14) [inline package.json]: @jenessawhite *react-native-vector-icons* is a peer dependency. No need to add it as a dependency

File: src/helpers/getIconType.js [hunk 2]
```diff
@@ -17,8 +16,6 @@ export default (type) => {
       return OcticonIcon
     case 'material':
       return MaterialIcon
-    case 'material-community':
```
Comment (binoy14) [inline src/helpers/getIconType.js]: @jenessawhite What's the reason for this change? If this is unrelated please revert it.

File: src/checkbox/CheckBox.js [hunk 3]
```diff
@@ -79,16 +79,6 @@ styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center'
   },
-  container: {
```
Comment (binoy14) [inline src/checkbox/CheckBox.js]: @jenessawhite What's the reason for this change? This seems to be useful for styling the component in line 22

**Comment by binoy14** [comment]
@jenessawhite Thanks for PR. Please review comments I made

File: src/helpers/getIconType.js (hunk 2 above)
Comment (jenessawhite) [inline src/helpers/getIconType.js]: i was using an older version where material-community was causing issues.

File: src/checkbox/CheckBox.js (hunk 3 above)
Comment (jenessawhite) [inline src/checkbox/CheckBox.js]: I removed this for my project but definitely can be useful for others.

**Comment by jenessawhite** [comment]
Thank you for the feedback. I'll fix those! Sorry, it's my first PR.

**Comment by jenessawhite** [comment]
@binoy14 should be all good now! Thanks

**Comment by binoy14** [comment]
@jenessawhite I will review it later. Let me know if u want to make more PRs there are lot of fun issues and I can help you if you need it.

**Comment by jenessawhite** [comment]
That'd be awesome. I'd really like that.

**Comment by Monte9** [comment]
@binoy14 This PR lgtm. Is it ready to be merged?

**Comment by binoy14** [comment]
@jenessawhite Look thru the issues and comment on something you would like to work on.

@Monte9 I will test it and if all good will merge it.

File: src/checkbox/CheckBox.js [hunk 4]
```diff
@@ -28,15 +28,15 @@ const CheckBox = ({component, checked, iconRight, title, center, right, containe
       ]}>
       <View style={[
         styles.wrapper,
-        right && {justifyContent: 'flex-end'},
```
Comment (binoy14) [inline src/checkbox/CheckBox.js]: @jenessawhite What's the reason for removing this?

**Comment by binoy14** [comment]
Besides the last comment everything looks good

File: src/checkbox/CheckBox.js (hunk 4 above)
Comment (jenessawhite) [inline src/checkbox/CheckBox.js]: was interfering with a delete icon I was trying to use in same row. fixed.

**Comment by jenessawhite** [comment]
@binoy14 I will do that this week. Thanks for offering to help me out

**Comment by AlmogRnD** [comment]
How is this done, I don't see the docs explaining this

**Comment by Monte9** [comment]
@AlmogRnD Yeah, def missing the docs. Let's talk about this over at the issue here: https://github.com/react-native-training/react-native-elements/issues/545
